translations/en.json

```diff
@@ -77,7 +77,7 @@
   "contact": "Contact",
   "contact_us": "Contact <1>Us</1>",
   "donate_subtitle": "Support Lemmy development",
-  "donate_desc": "We are able to develop Lemmy as an open source platform free of tracking and ads thanks to the generosity of our users. Once a year we ask you to consider donating to support our work. Financial security allows us to continue maintaining and improving the platform. If you’d like to make a one-time or recurring donation simply click the button below. Thank you for using Lemmy.<br/><br/>Nutomic and Dessalines, Lemmy Developers",
+  "donate_desc": "We provide Lemmy as free and open source platform without any tracking or advertising, and work every day to improve it. Yet we also need money to pay our bills and provide for our families. Only 2% of Lemmy users donate, so we need your donation to keep this model working. Thank you for helping to create a new form of social media.<br/><br/>Nutomic and Dessalines, Lemmy Developers",
```

Comment:
Some minor corrections:

We provide Lemmy as a free and open source platform, without any tracking or advertising, and work every day to improve it.

> Yet we also need money to pay our bills and provide for our families. Only 2% of Lemmy users donate, so we need your donation to keep this model working. 

I think I like the original better: "Once a year we ask you to consider donating to support our work. Financial security allows us to continue maintaining and improving the platform. If you’d like to make a one-time or recurring donation simply click the button below. Thank you for using Lemmy."

Overall I think I just like the original better.

Reply:
The problem with that is, it only asks people to donate "if they like". But donations are something we need in order to be able to work on Lemmy (at least I need them). So the language should account for that.

Reply:
Rewrote the text again.